tests/data_context/store/test_datasource_store.py

```diff
@@ -83,11 +87,22 @@ def _assert_serialized_datasource_configs_are_equal(
     if len(datasource_configs) <= 1:
         raise AssertionError("Must provide at least 2 datasource configs")
 
-    dict_config_serializer = DictConfigSerializer(schema=datasourceConfigSchema)
+    if serializers is None:
+        serializers = [DictConfigSerializer(schema=datasourceConfigSchema)] * (
+            len(datasource_configs) + 1
+        )
+    else:
+        if len(serializers) <= 1:
+            raise AssertionError("Must provide at least 2 datasource serializers")
+        if not len(datasource_configs) == len(serializers):
+            raise AssertionError(
+                "Must provide the same number of serializers as datasource configs"
+            )
```

Comment:
This is all just used internally in tests right? Do we need error checking like this here?

Reply:
True, just added reflexively. Probably not required but might be helpful when using this method to build new tests. I know I screwed it up when writing the tests.